### DISCUSSION TITLE
CASI and LiDAR habitat

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#76] [Planning consideration]
#### What is it

A habitat map derived from airborne data, specifically CASI and LIDAR data. The habitat map is a polygon shapefile showing site relevant habitat classes, for sites where data is available.

#### Why are we looking at it

Because this dataset allows us to identify the presence of different types of habitats through airbourne data, it makes it easier to make planning and land use decisions in a sensitive way to the site.

**Comment by kieranwint** [comment]
Accessing CASI and LiDAR habitat data through [data.gov.uk](https://ckan.publishing.service.gov.uk/dataset/casi-and-lidar-habitat-map/resource/cae8e658-51e0-4002-aa9c-d01302709d1c) as a WMS, 30 potential datasets are accessible, such as first return, last return, Digital Terrain Model (DTM), Digital Surface Model (DSM), 25cm, 1m, 2m, point cloud etc.
We need to do identify which dataset/s of these 30 would be suitable for the platform and the specified identified planning needs.
